README.md

```diff
@@ -1 +1,37 @@
-# angular-what-is-a-directive-readme
\ No newline at end of file
+# What is a Directive?
+
+## Overview
+
+You might not have noticed, but we've used directives already in our previous labs. Directives are essential to learning Angular, so hold in tight!
+
+## Objectives
+
+- Describe what Directives are
+
+## Directives, deep down
+
+Directives can be compared to components. They are essentially "markers" on a HTML element that tell Angular to attach a specific behaviour to the HTML element - either something small like a click event, or actually completely transforming the DOM node to display a list of data.
```

Comment:
students may not be familiar with components. I think the rest of this explanation stands up well enough of it's own without that comparison, so feel free to just remove it.